package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "react-movies-app",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "dev": "cross-env NODE_ENV=development webpack-dev-server --mode development --open",
```

Comment:
You should bear in mind that `cross-env` is [no longer developed](https://github.com/kentcdodds/cross-env/issues/257), which may be the reason of its [declined popularity](https://www.npmtrends.com/cross-env-vs-dotenv), so you may opt for [corresponding command-line parameter](https://webpack.js.org/api/cli/#node-env), consider alternative solution or expose [webpack config as a function](https://webpack.js.org/configuration/configuration-types/#exporting-a-function) and pass current [mode as a command line parameter](https://webpack.js.org/configuration/configuration-types/#exporting-a-function).